README.md

```diff
@@ -1,15 +1,27 @@
 # Awesome Web3 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/ethstorage/awesome-web3/)
 
-A curated list of awesome Web3 apps, libraries, tools and more.
+A list of awesome Web3 apps, NFTs, libraries, tools and more.
 
 - [Awesome Web3](#awesome-web3-awesomehttpscdnrawgitcomsindresorhusawesomed7305f38d29fed78fa85652e3a63e154dd8e8829mediabadgesvghttpsgithubcomethstorageawesome-web3)
```

Comment:
Why do we need this top-level title since the repo's name is ready Awesome Web3?